charts/base-cluster/templates/rbac/roleBindings.yaml

```
@@ -4,13 +4,14 @@
 {{- range $roleName, $roleMapping := $roles -}}
   {{- $clusterMapping := dig "clusterMapping" (dict) $roleMapping -}}
   {{- $namespaceMapping := dig "namespaceMapping" (dict) $roleMapping -}}
-  {{- $roleFullName := has $roleName $definedRoles | ternary (printf "%s-%s" (include "common.names.fullname" $) $roleName) $roleName -}}
+  {{- $roleBindingFullName := printf "%s-%s" (include "common.names.fullname" $) $roleName -}}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)